source/lex.tex

```diff
@@ -178,7 +178,7 @@
 \item
 Adjacent \grammarterm{string-literal} tokens are concatenated\iref{lex.string}.
 
-\item Whitespace characters separating tokens are no longer
+\item \grammarterm{whitespace-character}s separating tokens are no longer
```

Comment:
Starting a sentence with a (lowercase) grammar non-terminal feels odd.  Can we rephrase to avoid that?

Reply:
I could not find a good way to turn the sentence around, so just started with an extra word, "Any *whitespace-character*s separating..."